action/action_plugins/CMakeLists.txt

```diff
@@ -1,60 +1,26 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(action_plugins)
 
-add_compile_options(-std=c++14)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  pluginlib
-  scenario_actions
-  scenario_intersection
-  scenario_logger
-  scenario_utility
-)
-
-catkin_package(
-  INCLUDE_DIRS include
-  CATKIN_DEPENDS pluginlib
-                 scenario_actions
-                 scenario_intersection
-                 scenario_logger
-                 scenario_utility
-)
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
 include_directories(
-  include
-  ${CMAKE_CURRENT_SOURCE_DIR}
-  ${catkin_INCLUDE_DIRS}
-  )
-
-add_library(${PROJECT_NAME} SHARED
-  # src/acceleration_action.cpp
-  # src/change_signal_action.cpp
-  # src/follow_route_action.cpp
-  # src/lane_change_action.cpp
-  # src/speed_action.cpp
-  src/targets.cpp
+  include)
+
+ament_auto_add_library(${PROJECT_NAME} SHARED
+   src/acceleration_action.cpp
+   src/change_signal_action.cpp
+   src/follow_route_action.cpp
+   src/lane_change_action.cpp
+   src/speed_action.cpp
+#  src/targets.cpp
```

Comment:
I guess it because target.cpp only included the other files and you changed that? Then let's remove the file and the reference to it here

Reply:
Removed.